src/java.base/share/conf/security/java.policy

```diff
@@ -30,6 +30,8 @@ grant {
     permission java.util.PropertyPermission "line.separator", "read";
     permission java.util.PropertyPermission
                    "java.specification.version", "read";
+    permission java.util.PropertyPermission
+                   "java.specification.maintenance.version", "read";
```

Comment:
For consistency, you should probably also add a hard-coded permission to the `sun.security.provider.PolicyFile.initStaticPolicy()` method which is used as a fallback if there is a problem parsing the `java.policy` file. I doubt this will cause any issues as I don't suspect many/any applications will suddenly start reading this property and running with an SM but again it is mainly for consistency as all the other default permissions in this file are also granted in that method.

Reply:
Will do; thanks. Just to double-check, under the current proposal, this property will be undefined most of the time. I assume it is fine for the permissions to grant the ability to read a property that is not actually there.

Reply:
Yes.